src/com/itacademy/eshop/product/Books.java

```diff
@@ -0,0 +1,21 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.product.types.Category;
+
+public abstract class Books extends Product{

```

Comment:
Don't use plural in class name. Name books could be understood as list of Object Books. In this case you could just make this class not abstract, implement methods, move bookAuthor and bookCover fields in this class and delete Book class. Or if you want to keep this class, just change it's name to Book and change the current Book class to samething more specific. however it does make more sense if you just use one class since all books have author and cover, not just more specific ones